jenkins-vars.yml

```diff
@@ -5,7 +5,7 @@ project_name: docker-baseimage-alpine
 external_type: os
 release_type: stable
```

Comment:
the release type should be prerelease assuming 3.17/master will be the new stable, right?

Reply:
Oops, always forget that one.